packages/@aws-cdk/aws-rds/README.md

```diff
@@ -13,86 +14,126 @@
 ---
 <!--END STABILITY BANNER-->
 
-### Starting a Clustered Database
+```typescript
+import * as rds from '@aws-cdk/aws-rds';
+```
+
+### Starting a clustered database
 
 To set up a clustered database (like Aurora), define a `DatabaseCluster`. You must
 always launch a database in a VPC. Use the `vpcSubnets` attribute to control whether
 your instances will be launched privately or publicly:
 
 ```ts
-const cluster = new DatabaseCluster(this, 'Database', {
-    engine: DatabaseClusterEngine.AURORA,
-    masterUser: {
-        username: 'clusteradmin'
+const cluster = new rds.DatabaseCluster(this, 'Database', {
+  engine: rds.DatabaseClusterEngine.AURORA,
+  masterUser: {
+    username: 'clusteradmin'
+  },
+  instanceProps: {
+    // optional, defaults to the engine's default instance type
+    instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
+    vpcSubnets: {
+      subnetType: ec2.SubnetType.PRIVATE,
     },
-    instanceProps: {
-        instanceType: ec2.InstanceType.of(ec2.InstanceClass.BURSTABLE2, ec2.InstanceSize.SMALL),
-        vpcSubnets: {
-            subnetType: ec2.SubnetType.PRIVATE,
-        },
-        vpc
-    }
+    vpc,
+  },
 });
 ```
+
+To use a specific version of the engine
+(which is recommended, in order to avoid surprise updates when RDS add support for a newer version of the engine),
+use the static factory methods on `DatabaseClusterEngine`:
+
+```typescript
+new rds.DatabaseCluster(this, 'Database', {
+  engine: rds.DatabaseClusterEngine.defaultAurora({
```

Comment:
`defaultAurora` - what's a default aurora?

It's either mysql-compatible aurora or postgres-compatible aurora AFAIK.

From the [product page](https://aws.amazon.com/rds/aurora/) - "Amazon Aurora is a MySQL and PostgreSQL-compatible relational database built for the cloud"

Reply:
There are 3 supported engine types currently for Aurora: `aurora`, `aurora-mysql`, and `aurora-postgres`. This represents the first one.

I'm all ears if you have a better name.

Reply:
> I'm all ears if you have a better name.

`aurora`?

Reply:
From the docs,

> https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBInstance.html

> aurora (for MySQL 5.6-compatible Aurora)
> aurora-mysql (for MySQL 5.7-compatible Aurora)

Looks like aurora is just legacy name of aurora-mysql

Should we model these as a single engine but with both sets of versions merged, and do the right thing under the hood?

Reply:
> Should we model these as a single engine but with both sets of versions merged, and figure out the correct values under the hood?

That's not an option IMO - it would make it impossible to use one of those engine types (either `aurora` or `aurora-mysql`) without specifying the version.

Reply:
> > I'm all ears if you have a better name.
> 
> `aurora`?

Changed to `aurora` in the latest revision.